CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# dbt_fivetran_log v0.6.3
+## New Pricing Model Updates
+- Introduces a new macro `does_table_exist` as part of additional logic to `stg_fivetran_log__usage`. This checks and pulls from the `usage_cost` source table for customers on the new pricing model, which replaces the old `credits_used` table. Downstream models have also been updated to reflect the pricing changes. ([#50](https://github.com/fivetran/dbt_fivetran_log/pull/50))
```

Comment:
We should also add the fact that we renamed some fields and models. Can you add that to the CHANGELOG as well.